dts/msm8916/rules.mk

```diff
@@ -23,7 +23,6 @@ DTBS += \
 	$(LOCAL_DIR)/msm8916-mtp-smb1360.dtb \
 	$(LOCAL_DIR)/msm8916-512mb-qrd-skuh.dtb \
 	$(LOCAL_DIR)/msm8916-qrd-skut1.dtb \
-	$(LOCAL_DIR)/msm8916-qrd4-v1.dtb \
```

Comment:
Drop `msm8916-qrd4-v1.dtb`, and add `msm8916-512mb-qrd-skuh.dtb`, but keep the rest untouched.